src/main/webapp/styles/common.css

```diff
@@ -49,7 +49,7 @@ h3 {
 }
 
 p {
-  font-family: 'Libre Baskerville', serif;
+  font-family: 'Chelsea Market', cursive;
   font-size: 16px;
```

Comment:
It's recommended to use short-hands (https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties). Though I am not sure about the short-hand syntax. Could you please check?

Reply:
Great suggestion! However, since this change will make a lot of changes in the common.css (i.e. all the h1 h2 h3 etc), I think it might be better to change this in another branch, especially since common.css are used in all files. Please let me know what you think :)